src/SFA.DAS.AssessorService.Application.Api/Services/StandardImportService.cs

```diff
@@ -0,0 +1,117 @@
+﻿using SFA.DAS.AssessorService.Api.Types.Models.Standards;
+using SFA.DAS.AssessorService.Application.Interfaces;
+using SFA.DAS.AssessorService.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.AssessorService.Application.Infrastructure.OuterApi;
+
+namespace SFA.DAS.AssessorService.Application.Api.Services
+{
+    public class StandardImportService : IStandardImportService
+    {
+        private readonly IStandardRepository standardRepository;
+
+        public StandardImportService(IStandardRepository standardRepository)
+        {
+            this.standardRepository = standardRepository;
+        }
+        public async Task LoadStandards(IEnumerable<StandardDetailResponse> standards)
+        {
+            Func<StandardDetailResponse, Standard> MapGetStandardsListItemToStandard = source => new Standard
+            {
+                StandardUId = source.StandardUId,
+                IfateReferenceNumber = source.IfateReferenceNumber,
+                LarsCode = source.LarsCode,
+                Title = source.Title,
+                Version = source.Version,
+                Level = source.Level,
+                Status = source.Status,
+                TypicalDuration = source.TypicalDuration,
+                MaxFunding = source.MaxFunding,
+                IsActive = source.IsActive,
+                LastDateStarts = source.StandardDates?.LastDateStarts,
+                EffectiveFrom = source.StandardDates?.EffectiveFrom,
+                EffectiveTo = source.StandardDates?.EffectiveTo,
+                VersionApprovedForDelivery = source.VersionDetail.ApprovedForDelivery,
+                VersionEarliestStartDate = source.VersionDetail.EarliestStartDate,
+                VersionLatestEndDate = source.VersionDetail.LatestEndDate,
+                VersionLatestStartDate = source.VersionDetail.LatestStartDate,
+                ProposedMaxFunding = source.VersionDetail.ProposedMaxFunding,
+                ProposedTypicalDuration = source.VersionDetail.ProposedTypicalDuration
+            };
+
+            await standardRepository.DeleteAll();
```

Comment:
This would be better as a separate method as LoadStandards doesn't suggest it deletes them all first to me

Reply:
I've added this as a seperate method